src/Sensor/SensorBms.h

```diff
@@ -0,0 +1,90 @@
+#ifndef _SENSOR_BMS_H_
+#define _SENSOR_BMS_H_
+
+#include "CanListener.h"
+
+#define NUM_PARAMS                  7
+
+#define PARAM_ID_BATTERY_VOLTAGE    0x14
+#define PARAM_ID_BATTERY_CURRENT    0x15
+#define PARAM_ID_MAX_CELL_VOLTAGE   0x16
+#define PARAM_ID_MIN_CELL_VOLTAGE   0x17
+#define PARAM_ID_STATUS             0x18
+#define PARAM_ID_SOC                0x1A
+#define PARAM_ID_TEMP               0x1B
+
+#define STATUS_CHARGING             0x91
+#define STATUS_CHARGED              0x92
+#define STATUS_DISCHARGING          0x93
+#define STATUS_REGENERATION         0x96
+#define STATUS_IDLE                 0x97
+#define STATUS_FAULT_ERROR          0x9B
+
+using namespace can;
+
+class SensorBms : public CanListener {
+    public:
+
+        /**
+         * CONSTRUCTOR
+         */
+        SensorBms(CanInterface &canInterface, uint16_t requestIntervalMs);
+
+        /**
```

Comment:
Can you add comments for each method in the same style as the other header files for other sensors?